packages/angular/src/utils/versions.ts

```diff
@@ -28,5 +28,6 @@ export const tsNodeVersion = '10.9.1';
 export const jestPresetAngularVersion = '~14.1.0';
 export const typesNodeVersion = '18.16.9';
 export const jasmineMarblesVersion = '^0.9.2';
+export const analogVitestAngular = '1.7.0';
```

Comment:
Should this be pinned? 

Could we get away with `~1.7.0`?

Reply:
`~1.7.0` should definitely work.
That said, this is an interesting topic to discuss beyond this PR. I personally like the predictability of generators and migrations pinning the versions explicitly and totally relying on Nx to update them.
This means that we'd need something like a custom Renovate manager to pick the versions from this file (if it's not already there and I missed it)

Reply:
I understand that, but using a range allows for multiple things.

1. Our e2es will catch breaking changes if a package we depend on releases a version which breaks anything for us
2. New workspaces aren't locked down to an older version OTTB

Our migrations run based on specific Nx releases. If we pinned every version of every dependency, we'd need a lot more migrations and lot more releases of Nx more frequently to keep users up to date. There's a high maintenance tax to pay there, especially when a range of versions tends to be sufficient.

Reply:
Agreed! The plugin already received two patches since I opened this PR 😊

> Our e2es will catch breaking changes if a package we depend on releases a version which breaks anything for us
Not really. Some bugs might go through.
In that case, we would simply update the range.
The only challenge is that users might struggle with the repro as the same nx version might use different plugin versions depending on when the workspace is generated.

Anyway, this also depends on the stability of the package we are depending on. In this case, I am very confident about this one. The analog team is doing a great job 😉

Reply:
Done!